pkg/oc/admin/policy/subject_review.go

```diff
@@ -127,12 +127,12 @@ func (o *sccSubjectReviewOptions) Complete(f *clientcmd.Factory, args []string,
 	return nil
 }
 
-func (o *sccSubjectReviewOptions) Run(args []string) error {
+func (o *sccSubjectReviewOptions) Run(args []string, f *clientcmd.Factory) error {
```

Comment:
We should try to avoid passing the factory to the `Run` method. Take everything you need in `Complete`.  ;)